packages/init/bootstrap-starter/test.sh

```diff
@@ -5,4 +5,4 @@
 # and only runs the tests in the `near-runner` folder.
 cd near-runner
 npm install
-npm run test
\ No newline at end of file
+npm run test -- $@ # pass along any CLI flags, such as `--verbose`
```

Comment:
Do we like having this?

@ChaoticTempest do you know how to do this same thing with the `test.bat` script that's included?

Reply:
I believe it's `%*` instead of `$@` to pass arguments